src/test/java/teammates/test/cases/ui/pagedata/InstructorFeedbackResultsPageDataTest.java

```diff
@@ -0,0 +1,1584 @@
+        verifyKeysOfMap(singleSectionSectionPanels, Arrays.asList("Section 1"));
+        
+        ______TS("RQG single section case - verify section panel");
+        InstructorFeedbackResultsSectionPanel singleSectionSectionPanel = singleSectionSectionPanels.get("Section 1");
+        assertEquals("Section 1", singleSectionSectionPanel.getSectionNameForDisplay());
+        assertEquals("Detailed Responses", singleSectionSectionPanel.getDetailedResponsesHeaderText());
+
+        verifyKeysOfMap(singleSectionSectionPanel.getIsTeamWithResponses(), Arrays.asList("Team 1.1"));
+        assertEquals("panel-success", singleSectionSectionPanel.getPanelClass());
+        assertEquals("Received Responses Statistics", singleSectionSectionPanel.getStatisticsHeaderText());
+        
+        assertTrue(singleSectionSectionPanel.isDisplayingTeamStatistics()); // note that this is still true because
+                                                               // displaying stats are done in javascript
+      
+        Map<String, List<InstructorFeedbackResultsParticipantPanel>> singleSectionStudentPanel = singleSectionSectionPanel.getParticipantPanels();
+        InstructorFeedbackResultsGroupByQuestionPanel studentPanel = (InstructorFeedbackResultsGroupByQuestionPanel) singleSectionStudentPanel.get("Team 1.1").get(0);
+        List<InstructorFeedbackResultsQuestionTable> singleSectionQuestionTables = studentPanel.getQuestionTables();
+        assertEquals(2, singleSectionQuestionTables.size());
+        
+        ______TS("RQG single section case - verify question table");
+        InstructorFeedbackResultsQuestionTable singleSectionQuestionTable = singleSectionQuestionTables.get(0);
+        
+        assertEquals(1, singleSectionQuestionTable.getResponses().size());
+        assertEquals("", singleSectionQuestionTable.getAdditionalInfoText());
+        assertEquals("panel-info", singleSectionQuestionTable.getPanelClass());
+        assertEquals(Sanitizer.sanitizeForHtml(dataBundle.feedbackQuestions.get("qn1InSession1InCourse1").questionMetaData.getValue()), 
+                                        singleSectionQuestionTable.getQuestionText());
+        verifyHtmlClass(singleSectionQuestionTable.getResponsesBodyClass(), "panel-collapse", "collapse", "in");
+        
+        verifyColumns(singleSectionQuestionTable.getColumns(), "Photo", "Giver", "Team", "Feedback", "Actions");
+        verifyKeysOfMap(singleSectionQuestionTable.getIsColumnSortable(),
+                        Arrays.asList("Photo", "Giver", "Team", "Feedback", "Actions"));
+        verifyTrueValuesInMap(singleSectionQuestionTable.getIsColumnSortable(), 
+                               Arrays.asList("Giver", "Team", "Feedback"));
+        verifyFalseValuesInMap(singleSectionQuestionTable.getIsColumnSortable(), 
+                               Arrays.asList("Photo", "Actions"));
```

Comment:
Are there a limited number of these lists of strings?
Can they be extracted into properly named constants?

For `verifyTrueValuesInMap()` and `verifyFalseValuesInMap()`, are they always used in conjunction with the parameters for one being the opposite for the other?
Would it be a good idea to have one method that checks if a list of keys result in true and everything else result in false, or are there keys that belong to neither?